spec/API_specification/array_api/linalg.py

```diff
@@ -110,59 +110,82 @@ def diagonal(x: array, /, *, offset: int = 0) -> array:
     """
 
 def eigh(x: array, /) -> Tuple[array]:
-    """
-    Returns an eigendecomposition x = QLQᵀ of a symmetric matrix (or a stack of symmetric matrices) ``x``, where ``Q`` is an orthogonal matrix (or a stack of matrices) and ``L`` is a vector (or a stack of vectors).
+    r"""
+    Returns an eigenvalue decomposition of a complex Hermitian or real symmetric matrix (or a stack of matrices) ``x``.
+
+    If ``x`` is real-valued, let :math:`\mathbb{K}` be the set of real numbers :math:`\mathbb{R}`, and, if ``x`` is complex-valued, let :math:`\mathbb{K}` be the set of complex numbers :math:`\mathbb{C}`.
+
+    The **eigenvalue decomposition** of a complex Hermitian or real symmetric matrix :math:`x \in\ \mathbb{K}^{n \times n}` is defined as
+
+    .. math::
+       x = Q \Lambda Q^H
+
+    with :math:`Q \in \mathbb{K}^{n \times n}` and :math:`\Lambda \in \mathbb{R}^n` and where :math:`Q^H` is the conjugate transpose when :math:`Q` is complex and the transpose when :math:`Q` is real-valued and :math:`\Lambda` is a diagonal matrix whose diagonal elements are the corresponding eigenvalues. When ``x`` is real-valued, :math:`Q` is orthogonal, and, when ``x`` is complex, :math:`Q` is unitary.
```

Comment:
@kgryte regarding _"When ``x`` is real-valued, :math:`Q` is orthogonal, and, when ``x`` is complex, :math:`Q` is unitary."_, I'm wondering why the distinction. I tested with numpy and pytorch, and behavior seems to be the same. In this case the phrasing implies that eigenvectors for complex inputs are not orthogonal. For random inputs I do get orthogonal vectors. Not sure when that would not be the case.

Reply:
This does match the PyTorch docs, so I think it did have some review/discussion.

Reply:
For clarity, an orthogonal matrix is the real analogue of a unitary matrix.

Reply:
Ah right, linalg lingo:)